cmd/algod/main.go

```diff
@@ -289,7 +289,7 @@ func main() {
 		}
 	}
 
-	err = s.Initialize(cfg, phonebookAddresses)
+	err = s.Initialize(cfg, phonebookAddresses, string(genesisText[:]))
```

Comment:
probably don't need `[:]`
why not reference into `github.com/algorand/go-algorand/daemon/algod/api/server/lib` here and skip bouncing off daemon/algod/server.go ?

Reply:
Thanks, I thought I had tried that. I didn't want to add another import to main.go which is why I opted to pass it into the initialize function.